src/Dispatcher/AlertDispatcher.php

```diff
@@ -17,8 +18,15 @@ class AlertDispatcher
     public $dontAlertExceptions;
     public $notificationLevel;
 
+    /**
+     * AlertDispatcher constructor.
+     * @param Exception|Throwable $exception
+     * @param array $dontAlertExceptions
+     * @param array $notificationLevelsMapping
+     * @param array $exceptionContext
+     */
     public function __construct(
-        Exception $exception,
+        $exception,
```

Comment:
Is there any issue with just using `Throwable` here? `Exception` implements the interface since PHP 7, which is the minimum supported PHP version for this package. If that's changed, I'd remove the phpdoc as it doesn't add much value. All of the variables are already type hinted and there's no description provided in the phpdoc for each of the variables.

Reply:
Thank you for your review.
Yes `Throwable` is better. `Throwable` should working fine in previous Laravel 5 & 6 too

I also updated readme accordingly and vup to 5.0

Reply:
Thanks I ll merge and push after @ikari7789 's approval